### ISSUE TITLE
ofEasyCam distance not set until second frame

### ISSUE THREAD

**Issue by neilmendoza** [openframeworks/openFrameworks#5018]
At the moment, **ofEasyCam::update()** is not called until the second frame as events are not attached to the camera it until the first call of **ofEasyCam::begin()**. This is problematic as the camera distance is set in **ofEasyCam::update()**. This was causing problems for me as I expected the distance to be set up in the constructor so that I could use the camera in **ofApp::setup()** without needing to call **ofEasyCam::setDistance()** myself.

**Comment by arturoc** [comment]
yeah i've seen this before, the camera "jumps" to it's position after the first frame. it should be a matter of calling the update method internally from setup. do you want to send a PR?

**Comment by roymacdonald** [comment]
ofEasyCam doesn't have a setup function. 
I think that it is just a matter of moving[ these lines](https://github.com/openframeworks/openFrameworks/blob/master/libs/openFrameworks/3d/ofEasyCam.cpp#L50-L52) into the begin() function. Thus the distance will be set the first time begin gets called.  Haven't tested this though. 
Best!

**Comment by bakercp** [comment]
@roymacdonald do you have any thoughts on this as you've been working with ofEasyCam lately ...

**Comment by roymacdonald** [comment]
Hi.
I haven't seen this "jump" although I understand where it comes from.
moving [these lines](https://github.com/openframeworks/openFrameworks/blob/master/libs/openFrameworks/3d/ofEasyCam.cpp#L20-L22) into the begin method should fix this. I haven't tested it yet. 
Actually moving these lines makes sense as these are related to the view of the camera while what's handled by the update method are interaction related things.

I'll do so soon and make a PR.
